### PR TITLE
[SD-2910] Add simple package and picture publishing to NewsML G2

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

File: server/apps/publish/formatters/newsml_g2_formatter.py [hunk 1]
```diff
@@ -34,21 +35,35 @@ class NewsMLG2Formatter(Formatter):
...
         except Exception as ex:
             raise FormatterError.newmsmlG2FormatterError(ex, subscriber)
 
+    def _is_package(self, article):
+        """
+        Given an article returns if it is a none takes package or not
+        :param artcile:
+        :return: True is package
+        """
+        return article[ITEM_TYPE] == CONTENT_TYPE.COMPOSITE and article.get(PACKAGE_TYPE, '') == ''
```
Comment (mdhaman) [inline server/apps/publish/formatters/newsml_g2_formatter.py]: @marwoodandrew is this function only for the Takes Package a.k.a digital master file.

Comment (marwoodandrew) [inline server/apps/publish/formatters/newsml_g2_formatter.py]: Yes, poorly named you think? Only real packages need to be handled as packages, digital packages are handled as  news items.

**Pull request merged**